.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,44 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Bug Report
+description: File a bug report
+labels: ["Type: bug"]
+assignees: []
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug, including details regarding any error messages, version, and platform.
+      description: Please include what you expected.
+    validations:
+      required: true
+  - type: dropdown
+    id: component
+    attributes:
+      label: Component(s)
+      multiple: true
+      options:
```

Comment:
Do we actually need these `options`? Or should it be consistent with modules like `parquet-avro`, `parquet-cli`, etc.? Same question for other yamls.

Reply:
Also please note I introduced as many new labels as I could during migration. We now might want to reduce the current set by deleting them [here](https://github.com/apache/parquet-java/labels) (delete is irreversible). Same goes for [milestones](https://github.com/apache/parquet-java/milestones).

Reply:
Thanks for the feedback @gszadovszky. I've change the list like so:
```diff
      options:
-       - Java
        - Release
-       - Documentation
        - Continuous Integration
-       - Packaging
+       - Arrow
+       - Avro
+       - Pig
+       - Protobuf
+       - Scala
+       - Thrift
+       - CLI
        - Benchmarking
        - Other
```

> I think it makes more sense to somehow sum up Release+CI+Packaging.

Agreed, but I don't have a good suggestion how. For now I just removed packaging.

Reply:
Thank you, @rok!

It would be nice to see somehow how many issues we have related to `CI` and `Release`. Maybe `Other` is enough but not sure. I am fine as is, though.

One thing is clearly missing. This is the core functionality that most PRs have updates in. These are the modules `parquet-column`, `parquet-common`, `parquet-encoding`, `parquet-format-structures`, `parquet-generator`, `parquet-hadoop`. I am fine calling it `Core` if there are no better suggestion.

`parquet-hadoop` is definitely part of core currently, even though we want to get rid of the hadoop dependencies. After this work would be done, the separate component `Hadoop` would make sense, but currently I don't think we need it.

Reply:
To see distribution of component in Jira you can check the current (imported) label counts [here](https://github.com/apache/parquet-java/labels) - these are based on jira components, so CI and Release wont be present. For a quick idea of how much CI issues there are I made a quick dump of titles and bodies of issues: [issues_dump.csv](https://github.com/user-attachments/files/15985203/issues_dump.csv)

Reply:
Some quick searches in the csv suggests we have enough release and CI related issues to have them separated. So, I'm fine with the list as is, only the `Core` label is missing.

Reply:
Thanks @rok for working on this 🙌  I would be inclined to remove `Other` and make it optional. WDYT?

Reply:
Thanks for the feedback! :D I've pushed a change and here's the current list:
```markdown
- Core
- Release
- Continuous Integration
- Arrow
- Avro
- Pig
- Protobuf
- Scala
- Thrift
- CLI
- Benchmarking
```
I'm glad to defer to your suggestions as I'm quite unfamiliar with `parquet-java` (except for the migration).

Reply:
One more thing, I was looking at Iceberg regarding @gszadovszky's comment.

> I think it makes more sense to somehow sum up Release+CI+Packaging.

At Iceberg we've packaged these into a `Build` label.

Reply:
Seems like a good idea. Will do.

Reply:
Done.